src/model_signing/_serialization/file.py

```diff
@@ -110,8 +111,15 @@ def serialize(
             for future in concurrent.futures.as_completed(futures):
                 manifest_items.append(future.result())
 
+        rel_ignore_paths = [
+            pathlib.Path(os.path.relpath(p, model_path)) for p in ignore_paths
+        ]
+
         return manifest.Manifest(
-            model_path.name, manifest_items, self._serialization_description
+            model_path.name,
+            manifest_items,
+            self._serialization_description,
+            rel_ignore_paths,
```

Comment:
open question: should we consider ignore paths as part of the serialization description, since they affect how the model was serialized?

Reply:
I can probably move it there as well.